java/test/src/main/java/org/ray/api/test/KillActorTest.java

```diff
@@ -25,14 +39,54 @@ public boolean hang() throws InterruptedException {
     }
   }
 
-  public void testKillActor() {
+  @RayRemote
+  public static class KillerActor {
+
+    public void kill(RayActor<?> actor, boolean noReconstruction) {
+      Ray.killActor(actor, noReconstruction);
```

Comment:
I've merged the new API, can you rebase and update this?

Reply:
Done.